tests/output/DummyStreamWrapper.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class DummyStreamWrapper
+{
+    /** @var resource|null */
+    public $context;
+
+    /** @var resource|null */
+    public $handle;
+
+    public function stream_cast(int $castAs)
+    {
+        return false;
+    }
+
+    public function stream_close(): void { }
+
+    public function stream_open(string $path, string $mode, int $options = 0, ?string &$openedPath = null): bool
+    {
+        return true;
+    }
+
+    public function stream_read(int $count)
+    {
+        return false;
+    }
+
+    public function stream_seek(int $offset, int $whence = SEEK_SET): bool
+    {
+        return true;
+    }
+
+    public function stream_set_option(int $option, int $arg1, ?int $arg2): bool
+    {
+        return false;
+    }
+
+    public function stream_stat()
+    {
+        return [];
```

Comment:
I'm not sure that returning an empty array is fine; maybe return `false` instead.